packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -67,6 +67,14 @@ class DependencyExtractionWebpackPlugin {
 			if ( typeof this.options.requestToExternalModule === 'function' ) {
 				externalRequest =
 					this.options.requestToExternalModule( request );
+
+				// requestToExternalModule allows a boolean shorthand
+				if ( externalRequest === false ) {
+					externalRequest = undefined;
+				}
+				if ( externalRequest === true ) {
+					externalRequest = request;
+				}
```

Comment:
This fixes a small bug where we wouldn't run our default if we returned `false` (only available with modules).